File: game/scripts/vscripts/modifiers/upgrades/items/modifier_upgrade_item_broom_handle.lua
modifier_upgrade_item_broom_handle = class({})

function modifier_upgrade_item_broom_handle:RemoveOnDeath() return false end
function modifier_upgrade_item_broom_handle:IsPurgable() return false end

function modifier_upgrade_item_broom_handle:GetExtraRadius()
    return 95
end

function modifier_upgrade_item_broom_handle:GetTexture()
    return 'item_broom_handle'
end

File: src/vscripts/modifiers/upgrades/items/modifier_upgrade_item_broom_handle.ts
import { registerModifier } from "../../../lib/dota_ts_adapter";
import { ModifierUpgrade } from "../modifier_upgrade";

@registerModifier({ customNameForI18n: "modifier_upgrade_item_broom_handle" })
class ModifierItemBroomHandle extends ModifierUpgrade {
    GetExtraRadius() {
        return 95;
    }

    GetTexture() {
        return "item_broom_handle";
    }
}
